src/main/java/roomescape/domain/ThemeRanking.java

```diff
@@ -0,0 +1,30 @@
+package roomescape.domain;
+
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ThemeRanking {
+
+    private static final int RANKING_SIZE = 10;
+
+    private final Map<Theme, Integer> themeRankCount;
+
+    public ThemeRanking(List<Reservation> inRangeReservations) {
+        HashMap<Theme, Integer> calculateThemeRankCount = new HashMap<>();
```

Comment:
```suggestion
        Map<Theme, Integer> calculateThemeRankCount = new HashMap<>();
```
선언시 구현체가 아닌 인터페이스를 활용해보아요.

Reply:
수정 완료했습니다!!